index.test.js

```diff
@@ -19,6 +19,12 @@ describe('pr-lint-action', () => {
   const good_title_and_bad_branch = { title: '[PROJ-1234] a good PR title', ref_name: 'fix_things' }
   const bad_title_and_good_branch = { title: 'no ticket in me', ref_name: 'bug/PROJ_1234/a_good_branch' }
   const lower_case_good_title_and_branch = { title: '[proj-1234] a lower case good title', ref_name: 'bug/proj_1234/a_good_lowercase_branch' }
+  
+  const good_title_no_brackets = {title: 'PROJ-123 Fixed the issue', ref_name: 'unused'}
+  const good_multi_title_no_brackets = {title: 'PROJ-123, PROJ-4321 Fixed all the issues!', ref_name: 'unused'}
+  const bad_title_undefined_project = {title: 'NOPE-123 Introduced more issues', ref_name: 'unused'}
+  const good_multi_title_undefined_project = {title: 'PROJ-123, NOPE-123 You win some, you lose some', ref_name: 'unused'}
```

Comment:
will it work with [TT-1,TT-2] type of bracked tickets?

Reply:
[I've added a new test for it](https://github.com/Clarifai/pr-lint-action/pull/1/files#diff-275a6743b2ff7d68e3d76f3462691716R25), but yes it does. It's pretty loose and really only checks to make sure there's a TT-## of any sort at all. It doesn't check where it is in the title or how many there actually are, or if they're valid projects.

It just makes sure that at least one exists that references the projects defined in the [config file](https://github.com/Clarifai/clarifai/blob/master/.github/pr-lint.yml#L1-L2).